build.gradle

```diff
@@ -255,6 +257,11 @@ allprojects {
     */
     implementation group: 'xerces', name: 'xercesImpl', version: '2.12.2'
 
+    implementation group: 'org.apache.cxf', name: 'cxf-spring-boot-starter-jaxws', version: cxfVersion
+    implementation group: 'org.apache.cxf', name: 'cxf-rt-transports-http-jetty', version: cxfVersion
+//    implementation group: 'org.apache.cxf', name: 'cxf-rt-transports-http', version: cxfVersion
```

Comment:
Remove this commented out code if it isn't needed.

Reply:
done

Reply:
Confirmed commented out code removed.